projects/robots/epson/scaraEpsonT6/worlds/.industrial_example.wbproj

```diff
@@ -0,0 +1,11 @@
+Webots Project File version R2022b
+perspectives: 000000ff00000000fd0000000200000001000002a400000285fc0200000001fb0000001400540065007800740045006400690074006f00720100000016000002850000008900ffffff000000030000073800000145fc0100000001fb0000001a0043006f006e0073006f006c00650041006c006c0041006c006c0100000000000007380000006900ffffff000004920000028500000001000000020000000100000008fc00000000
```

Comment:
The default layout seems odd (too big node tree, too big console, too small 3d window)
![a](https://user-images.githubusercontent.com/44834743/154435853-a454f7ff-d044-4f70-9efb-6e1af5140315.png)